src/vue/components/nav/action-panel.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
       :id="config.rootElementId"
-      class="alpheios-action-panel alpheios-content"
+      :class="['alpheios-action-panel', 'alpheios-content', $store.state.actionPanel.showNav ? 'alpheios-action-panel--nav-visible' : '']"
```

Comment:
May be it would be better and easier readable to make it this way:
```
 class = "alpheios-action-panel alpheios-content"
:class = "{ 'alpheios-action-panel--nav-visible': $store.state.actionPanel.showNav}
```
Then it is easy to see static classes and conditional classes.
What do you think?

Reply:
That's a great idea! I somehow did not think about it before your suggestion. Updated.